source/_integrations/zha.markdown

```diff
@@ -273,7 +273,7 @@ ZHA support for binding and unbinding. Binding is an action in Zigbee which defi
 
 Binding of Zigbee clusters is nothing but establishing direct virtual connections between endpoints on different Zigbee nodes so that, during data transmission between the connected endpoints, the destination is identified by a specific cluster, rather than by the destination address. This is useful to for example bind a Zigbee end device like a remote or motion-sensor to a Zigbee lightbulb or group of lightbulbs. After this binding is done the remote or motion-sensor is attached directly to that device via a pair-to-pair connection and can direcly control that bound lightbulb / group of lightbulbs without going through the Zigbee coordinator. This means that the remote or motion-sensor can control the lightbulb  / group of lightbulbs even when the Zigbee coordinator is not available.
 
-By default ZHA bind remotes to the coordinator, so coordinator could receive ZCL commands from the remotes and originate zha_event's. However some remotes, like example Philips RWL021 can only be bound to a single destination and it is not possible to make this switch to bind to another destinations like a device or groups unless you first unbind the remote from the coordinator. After you unbind the remote from ZHA coordinator you can then bind it directly to any other Zigbee device or a group directly.
+Note that all devices support binding as it depends on the Zigbee implementation of the device itself. Also, by default ZHA bind remotes to the coordinator, so coordinator could receive ZCL commands from the remotes and originate zha_event's. However some remotes, like example Philips RWL021 can only be bound to a single destination and it is not possible to make this switch to bind to another destinations like a device or groups unless you first unbind the remote from the coordinator. After you unbind the remote from ZHA coordinator you can then bind it directly to any other Zigbee device or a group directly.
```

Comment:
I think this may be missing the word "not" and is in fact saying the opposite of the intended meaning.

Reply:
Indeed. Fixed now.